plugins/shapefilereader/pom.xml

```diff
@@ -32,9 +32,10 @@
   </licenses>
 
   <properties>
+    <mockito-core.version>5.11.0</mockito-core.version>
```

Comment:
If you use ${mockito.version} in the core/pom.xml, you will not need this property. Also, I believe we are using 5.10.0 instead of 5.11.0